### ISSUE TITLE
Bug: Adding a recipe with a quantity to an empty list changes that recipe's quantity in other lists

### ISSUE THREAD

**Issue by zbee** [PunishXIV/Artisan#40]
Reproduction:
- If you make a list (e.g. `Food (up to 99)`),
- add a recipe with sub-crafts with a quantity (e.g. `Baked Eggplant` x 33),
- make a new list by cloning that list (e.g. `Food (up to 180)`),
- delete the items in the list,
- add a recipe with sub-crafts with a quantity (e.g. `Baked Eggplant` x 60)

Then both lists will have the steps in it generated by the final step: crafting Baked Eggplant 60 times

The only way around this is to perform these steps in one of the list:
- delete the sub-crafts
- set the quantity of the main recipe to 1
- add the recipe with sub-crafts with a quantity
- reduce the main recipe by 1

Basically, you must leave behind a step, and if you don't then it edits The Original List + All Lists Cloned From It.

I suspect this has to do with the title for a list only appearing in the List Editor if you have a step in the list, and if you don't have any steps it appears that adding to the list defaults to a name for the list that is not being updated when you edit the name and is copied from The Original List, and that all Copied Lists still have in them.

**Comment by Taurenkey** [comment]
This is just a case of object orientated programming backfiring slightly. When you duplicate the list, the code copies the list by reference, so until the plugin is reloaded the two lists are essentially linked.

Thankfully, easily resolved, so will be addressed in the next update.
